src/js/providers/tracks-mixin.js

```diff
@@ -401,6 +401,18 @@ define(['../utils/underscore',
     ////// PRIVATE METHODS
     //////////////////////
 
+    function _addCueToTrack(track, vttCue) {
+        if (!utils.isEdge()) {
```

Comment:
ok

On Thu, Aug 18, 2016 at 2:21 PM, egreaves <notifications@github.com> wrote:

> In src/js/providers/tracks-mixin.js
> <https://github.com/jwplayer/jwplayer/pull/1376#discussion_r75361367>:
>
> > @@ -401,6 +401,18 @@ define(['../utils/underscore',
> >      ////// PRIVATE METHODS
> >      //////////////////////
> >
> > +    function _addCueToTrack(track, vttCue) {
> > +        if (!utils.isEdge()) {
>
> @hghazzi <https://github.com/hghazzi> TextTrackCue also exists in
> browsers with the VTTCue interface, so that wasn't a reliable check. We
> only need to use TextTrackCue in Edge. In IE11, we polyfill VTTCue and
> render with VTT.js.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/jwplayer/jwplayer/pull/1376/files/57f0bdbf66cf4189909dfc206589e543999d12c8#r75361367>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAjpdRXxnz7Cba2dWyEJtVbqV1LwI_7Kks5qhKK8gaJpZM4Jm0Pz>
> .
>